src/ru/nsu/ccfit/zuev/osu/OSUParser.java

```diff
@@ -70,6 +70,8 @@ public boolean openFile() {
             }
         } catch (IOException e) {
             Debug.e("OSUParser.openFile: " + e.getMessage(), e);
+        } catch (java.lang.NullPointerException e) {
+            Debug.e("OSUParser.openFile: " + e.getMessage(), e);
         }
```

Comment:
```suggestion
            Debug.e("OSUParser.openFile: " + e.getMessage(), e);
        } catch (NullPointerException e) {
            Debug.e("OSUParser.openFile: " + e.getMessage(), e);
        }
```
java.lang.NullPointerException -> NullPointerException